### PR TITLE
Create ADR 0006-transport-mulitlayer-with-webRTC.rst

### PR DESCRIPTION
**Pull request by fredo**
## Description

ADR for our transport decision to implement webRTC for short term improvements

### CODE REVIEW

File: docs/adr/0006-transport-mulitlayer-with-webRTC.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+there has been work done to abstract it to Raiden's needs. A customized solution can be "tailored" to adapt perfectly to raiden's use case and would probably be
+more lightweight than the current Matrix. A self-built transport of course requires additional maintainance capacity.
+
+
+
+Decision
+========
+
+Option 1 was decided to be a short term solution to improve performance and scalability. Matrix already facilitates easy connection establishment of
+WebRTC through special evetns which can be send via rooms. It seems to be a reasonable amount of effort to implement in comparison
```
Comment (Dominik1999) [inline docs/adr/0006-transport-mulitlayer-with-webRTC.rst]: Here WebRTC is written differently than one line below - webRTC

File: docs/adr/0006-transport-mulitlayer-with-webRTC.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
+Transport: Using Multilayer Transport with WebRTC
+=================
```
Comment (palango) [inline docs/adr/0006-transport-mulitlayer-with-webRTC.rst]: The underline is to short

File: docs/adr/0006-transport-mulitlayer-with-webRTC.rst (hunk 1 above)
Comment (palango) [inline docs/adr/0006-transport-mulitlayer-with-webRTC.rst]: Also s/evetns/events

File: docs/adr/0006-transport-mulitlayer-with-webRTC.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- Alice's opens a matrix room with Bob
+- Bob receives the invite and joins
+- Alice as well as Bob use a signalling server to receive webRTC
+- Both use the room and the special events to exchange canditates
+- Once they negotiated, they gonna open webRTC data channels outside of Matrix.
+- If the connection breaks, the private room between Alice and Bob still exists as a fallback and will be used immediately.
+
+
+Why webRTC as the above layer solution
+=====================================================
```
Comment (palango) [inline docs/adr/0006-transport-mulitlayer-with-webRTC.rst]: Underline to long

File: docs/adr/0006-transport-mulitlayer-with-webRTC.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+WebRTC is usually already built-in into any browser and very easy to use. As a matter of fact, the light client already implemented a 
+proof of concept. Additionaly, as mentioned above, Matrix already provides a supported channel negotiation for their VoIP.
+In some examples, WebRTC proved to realize subsecond mediated payments. Ideally with this solution, Matrix only serves for discovery and presence and can therefore accept a
+much higher number of users until it becomes the bottleneck.
+
+
+Current Status
+===============
```
Comment (palango) [inline docs/adr/0006-transport-mulitlayer-with-webRTC.rst]: also a tiny bit to long 👀

File: docs/adr/0006-transport-mulitlayer-with-webRTC.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+The implementation for the light client is almost merged.
```
Comment (palango) [inline docs/adr/0006-transport-mulitlayer-with-webRTC.rst]: Maybe make this a list?

**Pull request merged**